### PR TITLE
Split RepoEventMapper to make it more testable

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

**Review by krdabrowski — APPROVED** [review]

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,232 @@
+package org.alfresco.hxi_connector.live_ingester.messaging.in.mapper.property;
```
Comment (tpage-alfresco) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java:1]: Let's add the same copyright headers to test files too. If the build isn't failing then let's update the plugin to scan all code files.

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/utils/RepoEventTestUtils.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    {
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneId.of("UTC"));
+    }
+
+    public static void setType(RepoEvent<DataAttributes<NodeResource>> event, EventType type)
+    {
+        given(event.getType()).willReturn(type.getType());
+    }
+
+    public static void setNodeResource(RepoEvent<DataAttributes<NodeResource>> event, NodeResource nodeResource)
```
Comment (tpage-alfresco) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/utils/RepoEventTestUtils.java:38]: This method (and `setNodeResourceBefore`) need a bit of knowledge to use correctly. I'd suggest either or both of:
1. Add some javadoc to explain what they do and how to use them (e.g. `event.getData()` must return a mock).
2. Move the methods back into the test class where they are used (i.e. don't claim they're general purpose).

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java:27]: meh, I am using two different workspaces and I've set copyright up just on one of them :/ build is failing so is ok from this side

Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/mapper/property/PropertiesMapperTest.java:27]: fixed

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/utils/RepoEventTestUtils.java (hunk 2 above)
Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/utils/RepoEventTestUtils.java]: I am moving these methods to individual test classes then

**Review by tpage-alfresco — APPROVED** [review]
LGTM! :+1:

**Pull request merged**